### DISCUSSION TITLE
Mark queries in Data config if used in screen/DP

### DISCUSSION THREAD

**Discussion by vaskoZGU** [Budibase/budibase#12418] [Ideas]
In applications with many queries, it would be helpful if those used in screens/DPs had a label like "in use".
Similar to Portainer, which shows the images in use.

**Comment by ConorWebb96** [comment]
Hey @vaskoZGU,

I do like this idea. 

I think this may be better as feature request on Github issues instead of a discussion.

**Reply by melohagan** [reply]
There's a similar issue raised here: https://github.com/Budibase/budibase/issues/9686

It would be nice!

**Comment by vaskoZGU** [comment]
@melohagan  Thanks, that's exactly what I was thinking too.
